### PR TITLE
Add customer name column to CSV export

### LINKED ISSUE #6534: Downloads: Emailed Customer names missing on Orders Report
Customer names were missing on analytics order report when the date range is big like a year.
sample of 1 month report
[orders_2021-03-05_page-wc-admin_path--analytics-orders.xlsx](https://github.com/woocommerce/woocommerce/files/6088533/orders_2021-03-05_page-wc-admin_path--analytics-orders.xlsx)

and sample of 1 year report
[wc-orders-report-export-16149203382307.xlsx](https://github.com/woocommerce/woocommerce/files/6088538/wc-orders-report-export-16149203382307.xlsx)

### PR DESCRIPTION
**Pull request by hsingyuc**
Fixes #6534 

Add customer name column to CSV export

### Screenshots
Before
螢幕快照 2021-03-09 下午8 34 22

After
螢幕快照 2021-03-09 下午8 34 01

### Detailed test instructions:

-   Create more than 25 orders
-   Go to Analytics -> Orders -> Click "Download"
-   Click download link in the email
-   See customer column with customer full name

### CODE REVIEW

File: src/API/Reports/Orders/Controller.php [hunk 1]
```diff
@@ -475,6 +475,16 @@ public function get_collection_params() {
 		return $params;
 	}
 
+	/**
+	 * Get customer name column export value.
+	 *
+	 * @param array $customer Customer from report row.
+	 * @return string
+	 */
+	protected function _get_customer_name( $customer ) {
```
Comment (becdetat) [inline src/API/Reports/Orders/Controller.php:484]: We're still supporting PHP 7.0.0, which doesn't support specifying the visibility of a function. You also don't need to indicate visibility using a prefixed underscore - so `function get_customer_name( $customer )` should be fine here.

**Comment by becdetat** [comment]
This tests well, and other than the comment on the function visibility the code looks good too.

File: src/API/Reports/Orders/Controller.php (hunk 1 above)
Comment (becdetat) [reply src/API/Reports/Orders/Controller.php:484]: Sorry! I misread the PHP documentation, the visibility of a function is fine to use. If you drop that underscore it should be fine to merge.

**Comment by hsingyuc** [comment]
@becdetat  Thank you for the feedback. The changes have been pushed.

**Review by becdetat — APPROVED** [review]
Excellent, thanks @hsingyuc

**Pull request merged**